pkgs/unified_analytics/example/sample_rate.dart

```diff
@@ -7,15 +7,15 @@ import 'package:unified_analytics/src/utils.dart';
 /// The purpose of this example file is to demonstrate the sampling
 /// rate functionality from the survey handler.
 ///
-/// It defines a [remoteUniqueId] that subs in for a real survey's unique
```

Comment:
Seems like the lint removes some of the brackets from the dartdocs while keeping them in other places, was that intended? @devoncarew

Reply:
Yes - this is due to adding the `comment_references` lint to the latest version of `package:dart_flutter_team_lints`. We're now validating that dartdoc references can be resolved.

Where these resolved - or I could update the reference to point to a valid symbol - I kept the square brackets. Other places I removed the brackets or changed them to just use regular code formatting (back-ticks).

Reply:
Here, we're not able to refer to local variable references within the method.

Reply:
Ah got it, that's a very helpful lint! LGTM